types/kotlin2/WebScreen.kt

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (c) 2021 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.analytics.plan
+
+import im.vector.app.features.analytics.itf.VectorAnalyticsEvent
+
+// GENERATED FILE, DO NOT EDIT. FOR MORE INFORMATION VISIT
+// https://github.com/matrix-org/matrix-analytics-events/
+
+/**
+ * Triggered when the user changed screen on Element Web/Desktop
+ */
+data class WebScreen(
+        val $current_url: $current_url,
```

Comment:
This is not valid Kotlin code.
I will just (temporarily) delete this file on our side (in https://github.com/vector-im/element-android/pull/5223), but we should definitely validate the code from here by trying to build the generated file.
Why is the format different than the for the other files?